package.json

```diff
@@ -1,29 +1,34 @@
 {
-    "name": "degjs-easing",
-    "version": "1.0.2",
+    "name": "@degjs/easing",
+    "version": "2.0.0",
     "description": "Animation easing functions",
     "keywords": [
-    	"easing",
-    	"animation",
+        "easing",
+        "animation",
         "DEGJS"
     ],
     "author": {
         "name": "Ryan Heap",
         "email": "rheap@degdigital.com"
     },
+    "files": [
+        "src"
+    ],
     "main": "src/easing.js",
     "license": "MIT",
     "repository": {
         "type": "git",
         "url": "https://github.com/DEGJS/easing.git"
     },
     "bugs": "https://github.com/DEGJS/easing/issues",
-    "jspm": {
-        "registry": "jspm",
-        "format": "es6",
-        "main": "easing.js",
-        "directories": {
-            "lib": "src"
-        }
-    }
+    "devDependencies": {
+        "@babel/core": "^7.0.0",
+        "@babel/preset-env": "^7.0.0",
+        "@babel/register": "^7.0.0",
+        "jasmine": "^3.2.0"
+    },
+    "scripts": {
+        "test": "jasmine"
+    },
+    "dependencies": {}
```

Comment:
If we're publishing this to NPM and we're publishing this as a scoped module, you will need
`
"publishConfig": {
	"access": "public"
}
`
Added to this file so that it won't think you're scoped module is private

Reply:
Good catch, thanks